### PR TITLE
[typo] fix typo in comments & const value(SignalUsr2)

### PR DESCRIPTION
**Pull request by jxd134**

### CODE REVIEW

**Review by megaeasex — APPROVED** [review]
[TASK:easegress-pr-test SUCCESS]megaease/easegress Pull Request 181 Deploy Test Success

File: pkg/context/httpcontext.go [hunk 1]
```diff
@@ -317,7 +317,7 @@ func (ctx *httpContext) Log() string {
 		strings.Join(ctx.tags, " | "))
 }
 
-// Template returns HTTPTemplate rely interface
+// Template returns HTTPTemplate rely on interface
```
Comment (localvar) [inline pkg/context/httpcontext.go:320]:
```suggestion
// Template returns the template engine
```

Comment (jxd134) [reply pkg/context/httpcontext.go:320]: ok, keep it simple

**Review by megaeasex — APPROVED** [review]
[TASK:easegress-pr-test SUCCESS]megaease/easegress Pull Request 181 Deploy Test Success

File: pkg/context/httpresponse.go [hunk 2]
```diff
@@ -82,7 +82,7 @@ func (w *httpResponse) SetBody(body io.Reader) {
 	w.body = body
 }
 
-// None uses it currently, keep it for future maybe.
+// OnFlushBody None uses it currently, keep it for future maybe.
```
Comment (benja-wu) [inline pkg/context/httpresponse.go:85]:
```suggestion
// OnFlushBody adds an HTTP body flushing handler function
```

Comment (benja-wu) [reply pkg/context/httpresponse.go:85]: This function has been used in `MemoryCache`, so the original comment is outdated.

Comment (jxd134) [reply pkg/context/httpresponse.go:85]: Does easegress use OnFlushBody  function rigt now?
I confused use it on partial data flush to cliet conn.

Comment (jxd134) [reply pkg/context/httpresponse.go:85]: Got it.

Comment (benja-wu) [reply pkg/context/httpresponse.go:85]: * The short answer is Yes, check it out [here](https://github.com/megaease/easegress/blob/main/pkg/filter/proxy/proxy.go#L328)
* In the `response.go`, the comment is ` OnFlushBody registers a callback function on flush body` for `MockedHTTPResponse.OnFlushBody`, maybe we can align it with `MockedHTTPResponse`'s.

**Review by megaeasex — APPROVED** [review]
[TASK:easegress-pr-test SUCCESS]megaease/easegress Pull Request 181 Deploy Test Success

File: pkg/context/httpresponse.go (hunk 2 above)
Comment (jxd134) [reply pkg/context/httpresponse.go]: I will learn it in the future.

File: pkg/util/texttemplate/texttemplate.go [hunk 3]
```diff
@@ -78,7 +78,7 @@ type TemplateEngine interface {
 	// SetDict adds a temaplateRule and its value for later rendering
 	SetDict(template string, value interface{}) error
 
-	// GetDict returns the template rely dictionary
+	// GetDict returns the template rely on dictionary
```
Comment (benja-wu) [inline pkg/util/texttemplate/texttemplate.go:81]:
```suggestion
	// GetDict returns the template's dictionary
```

Comment (benja-wu) [reply pkg/util/texttemplate/texttemplate.go:81]: Let's also keep it simple here. :-)

**Review by megaeasex — APPROVED** [review]
[TASK:easegress-pr-test SUCCESS]megaease/easegress Pull Request 181 Deploy Test Success

**Review by benja-wu — APPROVED** [review]
LGTM

**Comment by haoel** [comment]
@jxd134 

Thanks for this PR.

And we have another misspelled word found by https://goreportcard.com/report/github.com/megaease/easegress

Could you please help correct them in this PR?

```
easegress/pkg/util/jmxtool/agent_controller_test.go
Line 128: warning: "shoudl" is a misspelling of "should" (misspell)
Line 132: warning: "shoudl" is a misspelling of "should" (misspell)
```

**Comment by jxd134** [comment]
> 
> 
> @jxd134
> 
> Thanks for this PR.
> 
> And we have another misspelled word found by https://goreportcard.com/report/github.com/megaease/easegress
> 
> Could you please help correct them in this PR?
> 
> ```
> easegress/pkg/util/jmxtool/agent_controller_test.go
> Line 128: warning: "shoudl" is a misspelling of "should" (misspell)
> Line 132: warning: "shoudl" is a misspelling of "should" (misspell)
> ```

no porblem

**Review by megaeasex — APPROVED** [review]
[TASK:easegress-pr-test SUCCESS]megaease/easegress Pull Request 181 Deploy Test Success

**Review by localvar — APPROVED** [review]

**Pull request merged**